nbgrader/converters/base.py

```diff
@@ -85,6 +87,24 @@ class BaseConverter(LoggingConfigurable):
     def _permissions_default(self) -> int:
         return 664 if self.coursedir.groupshared else 444
 
+    @validate('pre_convert_hook')
+    def _validate_pre_convert_hook(self, proposal):
+        value = proposal['value']
+        if isinstance(value, string_types):
+            value = import_item(self.pre_convert_hook)
+        if not callable(value):
+            raise TraitError("pre_convert_hook must be callable")
+        return value
+
+    @validate('post_convert_hook')
+    def _validate_post_convert_hook(self, proposal):
+        value = proposal['value']
+        if isinstance(value, string_types):
```

Comment:
nbgrader is python 3 only these days, so I think you can just check against str here (and above as well).